### PR TITLE
Use proxyquire to mock fs object ...

### PR DESCRIPTION
**Pull request by ta2edchimp**
Supersedes #7

### CODE REVIEW

**Comment by kentcdodds** [comment]
Somehow this causes `bin` to fall out of coverage reporting... Will look into this before merging...

File: test/bin.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import test from 'ava'
+
+test.todo('test bin later')
+
+try {
+  require('./bin') // requiring now for coverage until this is tested
```
Comment (kentcdodds) [inline test/bin.js]: Could you please update your PR to `require('../bin')` :-) That'll fix the coverage issue :-) Thanks!

Comment (ta2edchimp) [inline test/bin.js]: Done. Pushes the percentage of covered statements und lines just above the set threshold :)

Comment (kentcdodds) [inline test/bin.js]: Perfect

**Comment by kentcdodds** [comment]
Wanna add yourself to the contributors? https://github.com/kentcdodds/eslint-find-new-rules/blob/master/.all-contributorsrc [more info](https://github.com/kentcdodds/all-contributors)

**Comment by kentcdodds** [comment]
I've given you write access to the repo. Please keep doing PRs (and I will too) rather than committing directly to master :-)

**Pull request merged**